modules/powering-off-bare-metal-hosts-web-console.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="powering-off-bare-metal-hosts-web-console_{context}"]
+= Powering off bare-metal hosts
+
+You can power off bare-metal cluster hosts in the web console or by applying a patch in the cluster by using the OpenShift CLI (`oc`).
+Before you power off a host, you should mark the node as unschedulable and drain all pods and workloads from the node.
+
+.Prerequisites
+* You have installed a {op-system} compute machine on bare-metal infrastructure for use in the cluster.
+* You have logged in as a user with `cluster-admin` privileges.
+* You have configured the host to be managed and have added BMC credentials for the cluster host.
+You can add BMC credentials by applying a `Secret` custom resource (CR) in the cluster or by logging in to the web console and configuring the bare-metal host to be managed.
+
+.Procedure
+. In the web console, mark the node that you want to power off as unschedulable. Perform the following steps:
+
+.. Navigate to *Nodes* and select the node that you want to power off. Expand the *Actions* menu and select *Mark as unschedulable*.
+
+.. Manually delete or relocate running pods on the node by adjusting the pod deployments or scaling down workloads on the node to zero.
+Wait for the drain process to complete.
+
+. Navigate to *Compute* -> *Bare Metal Hosts*.
+
+. Expand the *Options menu* for the bare-metal host that you want to power off, and select *Power Off*.
+Select *Immediate power off*.
+
+. Alternatively, you can patch the `BareMetalHost` resource for the host that you want to power off by using `oc`.
```

Comment:
I wonder what you think of making this a substep of 3 since its an alternative to that step?